modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+In some circumstances, when a table was followed immediately by a text paragraph, and the user deleted the text paragraph by back-spacing through each character, pressing the backspace character after the last character was removed resulted in the insertion point moving in to the table. If the user continued pressing the backspace key, material in the table’s bottom-right cell was deleted.
+
+The expected behavior when backspacing on an empty line immediately following a table is for the insertion point to not move at all.
+
+With this update, {productname} behaves as expected. Pressing the backspace key on an empty line immediately following a table produces no change to the insertion point location.
+
+
+=== Text alignments, such as flush left and centered, could not be applied to `<pre>` elements
+
+Previously, {productname} did not support changing the alignment of text set within `+<pre>+` tags. However, if text that was set within other tags (for example, `+<p>+` tags or a heading tag, such as `+<h1>+`) had an alignment set and the text was then changed to be set within `+<pre>+` tags, the previously applied `align` style remained in place.
+
+With this update, {productname} now supports applying, altering and removing `align` styles to text set in `+<pre>+` tags.
+
+
+=== Nuget packages for .NET and .NET Core are now configured to copy TinyMCE into `/wwwroot/lib/` when TinyMCE is installed into a project
+
+Previously, the {productname} Nuget package did not automatically copy {productname} in to `/wwwroot/lib/` when installing {productname} into a .NET project.
+
+This resulted in {productname} not working as expected (or at all).
+
+This error has been corrected with this update. Installing the {productname} Nuget package into a .NET project automatically places {productname} in `/wwwroot/lib`, as expected.
+
+
+=== UI components, such as dialogs, would in some cases cause the esc keyup event to incorrectly trigger inside the editor
+
+In {productname} the *esc* (Escape) key can be used to exit or ‘escape’ from an active user-interface component. If a menu is open, pressing the esc key closes the menu. If a dialog is present, pressing the esc closes the dialog.
+
+{productname} responds to the `KeyDown` event and exits the active element when the *esc* key is pressed. Previously, however, the consequent `KeyUp` event (generated when the user stops pressing the *esc* key) was propogated to the {productname} editor. This should not have happened.
+
+The `KeyDown` event does not propagate outside the component listening for it and neither should the `KeyUp` event.
+
+As of this update, this is the behavior: pressing the *esc* key closes or exits an active user-interface component and neither the `KeyDown` or `KeyUp event are propogated outside the component listening for these events.
+
+
+=== Selection direction was not stored/restored when getting/setting selection bookmarks
```

Comment:
```suggestion
=== Selection direction was not stored or restored when getting or setting selection bookmarks
```
Per the style guide we shouldn't be using `/` so just correcting that to match the fixes we did in the TinyMCE changelog.